op-conductor/conductor/service.go

```diff
@@ -766,16 +820,29 @@ func (oc *OpConductor) stopSequencer() error {
 	//
 	// To that end we allow to cancel the StopSequencer call if we're shutting down.
 	latestHead, err := oc.ctrl.StopSequencer(oc.shutdownCtx)
-	if err == nil {
-		// None of the consensus state should have changed here so don't log it again.
-		oc.log.Info("stopped sequencer", "latestHead", latestHead)
-	} else {
+	if err != nil {
 		if strings.Contains(err.Error(), driver.ErrSequencerAlreadyStopped.Error()) {
 			oc.log.Warn("sequencer already stopped", "err", err)
 		} else {
+			// If we can't stop the sequencer, maintain consistency by not touching Rollup boost
 			return errors.Wrap(err, "failed to stop sequencer")
 		}
+	} else {
+		oc.log.Info("stopped sequencer", "latestHead", latestHead)
+	}
+
+	// If we get here, sequencer is successfully stopped, now disable Rollup boost
+	if oc.rollupBoost != nil {
+		if err := oc.rollupBoost.SetExecutionMode(oc.shutdownCtx, client.ExecutionModeDisabled); err != nil {
+			oc.log.Error("failed to disable rollup boost execution mode", "err", err)
+			// this is a critical error, we should return an error and trigger an alert
+			// to preserve strong consistency, we should not continue to sequence
+			// this is treated same as failed to stop sequencer
```

Comment:
I think we can afford rollupBoost disable failure? In this case, as long as sequencer is stopped, there will not be any ForkChoiceUpdate to builder to trigger block building.

The question here is that do we have any other unintended consequences of having potentially 2 rollupboost in execution mode enabled?

Reply:
1. The error here will lead to retry, and ideally retry would work
2. even if it does not work, we should monitor the error, and manually fix it

Reply:
Prefer the style of

```
if oc.rollupBoost == nil {
    return
}

normal logic
```

this could help reduce nesting

Reply:
> do we have any other unintended consequences of having potentially 2 rollupboost in execution mode enabled?

there can.  for example due to this [issue](https://github.com/ethereum-optimism/optimism/issues/15393) with remote builders (even ~5ms RTT latency will do) the race-condition can happen b/w active vs. passive sequencers) causing the builder jittering b/w `N`, `N-1` and `N-2` unsafe block height.  in particular we needed to open this [issue](https://github.com/paradigmxyz/reth/issues/15086) on `reth` and actively deduplicate the FCUs with [this](https://github.com/flashbots/bproxy/blob/v0.0.48/proxy/proxy_authrpc.go#L181-L207) until all issues are solved.